### PR TITLE
fixed the model saving bug

### PR DESCRIPTION
**Pull request by mitanshudodia**

### CODE REVIEW

File: customer-churn/main.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
+from mlfoundry import ModelFramework
```
Comment (chiragjn) [inline customer-churn/main.py:1]:
```suggestion
from truefoundry.ml import ModelFramework
```

Comment (akashg3627) [inline customer-churn/main.py:1]: please use import from truefoundry.ml

File: customer-churn/main.py [hunk 2]
```diff
@@ -17,12 +19,17 @@ def experiment_track(model, params, metrics):
     mlf_run.log_params(params)
     # log the metrics
     mlf_run.log_metrics(metrics)
+
+    # dump the model and then save it.
+    path = dump(model, "classifier.joblib")
```
Comment (akashg3627) [inline customer-churn/main.py:24]: should this be hardcoded only?

File: customer-churn/main.py [hunk 3]
```diff
@@ -1,13 +1,15 @@
+from truefoundry.ml import ModelFramework
 import pandas as pd
 from sklearn.metrics import accuracy_score, f1_score, precision_score, recall_score
 from sklearn.model_selection import train_test_split
 from sklearn.neighbors import KNeighborsClassifier as Classification
 from truefoundry.ml import get_client
```
Comment (akashg3627) [inline customer-churn/main.py:6]:
```suggestion
import pandas as pd
from sklearn.metrics import accuracy_score, f1_score, precision_score, recall_score
from sklearn.model_selection import train_test_split
from sklearn.neighbors import KNeighborsClassifier as Classification
from truefoundry.ml import get_client, ModelFramework
```

**Review by chiragjn — APPROVED** [review]

**Pull request merged**